src/Appwrite/Resque/Worker.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Appwrite\Resque;
+
+use Swoole\Runtime;
+
+use function Swoole\Coroutine\run;
```

Comment:
Why do we need this 2 statements?

Reply:
![Bildschirmfoto 2021-06-11 um 16 33 33](https://user-images.githubusercontent.com/1759475/121703145-d034db80-cad2-11eb-894d-2611bb1e1a6d.png)
hehe, you tell me 😄 

Actually I don't think they have any reason to be there 👍🏻  gonna remove them.